packages/console/src/monaco/MonacoUtils.ts

```diff
@@ -207,9 +235,11 @@ class MonacoUtils {
       .forEach(languageParam => {
         const language = languageParam;
         log.debug2('Overriding default language loader:', language.id);
-        language.loader = () => ({
-          then: () => {},
-        });
+
+        // @ts-ignore
+        // language.loader = () => ({
+        //   then: () => undefined,
+        // });
```

Comment:
If this isn't needed (console still works, tests work), then just remove these lines.